okio/src/commonMain/kotlin/okio/Path.kt

```diff
@@ -249,6 +280,12 @@ class Path private constructor(
     return buffer.toPath(directorySeparator = slash)
   }
 
+  override fun compareTo(other: Path): Int {
+    val bytesResult = bytes.compareTo(other.bytes)
+    if (bytesResult != 0) return bytesResult
+    return slash.compareTo(other.slash)
```

Comment:
Comparing slashes is cheaper, maybe switch the order of operations?

Reply:
But also more likely to almost always be true, so worth deferring until after you check bytes.

Reply:
I wanna order ’em that way to get this order:

`a/b`
`a\b`
`c/d`
`c\d`